packages/react-vapor/src/components/sideNavigation/SideNavigationMenuSection.tsx

```diff
@@ -1,63 +1,86 @@
+    svgName?: string;
+    svgClass?: string;
+    customIcon?: React.ReactNode;
+    onClick?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+}
 
-    render() {
-        return (
-            <div className="block navigation-menu-section">
-                {this.buildHeader()}
-                <SlideY in={!(this.props.expandable && !this.props.expanded)} timeout={350}>
-                    <div className="navigation-menu-section-items">{this.props.children}</div>
-                </SlideY>
-            </div>
+const SideNavigationHeader: React.FunctionComponent<SideNavigationHeaderProps> = (props) => {
+    let icon: React.ReactNode = null;
+    if (props.customIcon) {
+        icon = props.customIcon;
+    } else if (props.svgName) {
+        icon = (
+            <Svg
+                svgName={props.svgName}
+                svgClass={classNames(
+                    'navigation-menu-section-header-icon icon mod-lg transparency-3 fill-white',
+                    props.svgClass
+                )}
+            />
         );
+    } else {
+        icon = <span className="navigation-menu-section-header-icon" />;
     }
 
-    private buildHeader(): JSX.Element {
-        if (this.props.header) {
-            let arrow = null;
-            if (this.props.expandable) {
-                arrow = this.props.expanded ? (
-                    <Svg svgName="arrow-top-rounded" className={arrowClassName} />
-                ) : (
-                    <Svg svgName="arrow-bottom-rounded" className={arrowClassName} />
-                );
-            }
-            return (
-                <SideNavigationHeader {...this.props.header} onClick={() => this.handleClick()}>
-                    {arrow}
-                </SideNavigationHeader>
-            );
-        }
-        return <SideNavigationLoadingHeader />;
-    }
-}
+    return !_.isEmpty(React.Children.map(props.children, _.identity)) ? (
+        <div className="navigation-menu-section-header text-white" onClick={props.onClick}>
+            {icon}
+            {props.children}
+        </div>
+    ) : null;
+};
+
+export const SideNavigationMenuSection: React.FunctionComponent<ISideNavigationSectionProps> = ({
+    expandable,
+    expanded,
+    onClick,
+    header,
+    children,
+    title,
+    ...headerProps
+}) => {
+    const sectionHeader = (
+        <SideNavigationHeader {..._.extend({}, header, headerProps)} onClick={expandable ? _.noop : onClick}>
+            {title || (header && header.title)}
+        </SideNavigationHeader>
+    );
+    const items = <div className="navigation-menu-section-items">{children}</div>;
+
+    return expandable ? (
+        <Collapsible
+            className="navigation-menu-section"
+            id={_.uniqueId('nav-section')}
+            headerContent={sectionHeader}
+            toggleIconClassName="fill-white transparency-3"
+            onToggleExpandedState={onClick}
+            expanded={!!expanded}

```

Comment:
`expanded` is already a `boolean`, do you really need this `!!`?

Reply:
This is me being paranoid, expanded can be `undefined` instead of false. It doesn't do much of a difference except in the unit tests.